timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -103,6 +106,7 @@ def __init__(
         verbosity: int = 2,
         prediction_length: int = 1,
         quantile_levels: Optional[List[float]] = None,
+        early_stopping_patience: Optional[int] = None,
```

Comment:
similar to comment on learners, this should be provided during `fit` time and within `hyperparameters`.

Reply:
Will change it in the next revision.